content/v2.0/monitor-alert/send-email.md

```diff
@@ -0,0 +1,235 @@
+        data: bytes(v: "{
+          \"personalizations\": [{
+            \"to\": [{
+            \"email\": \”jane.doe@example.com\"}],
+            \"subject\": \”InfluxData critical alert\"
+            }],
+            \"from\": {\"email\": \"john.doe@example.com\"},
+            \"content\": [{
+              \"type\": \"text/plain\",
+              \"value\": \”Example alert text\"
+            }]
+        }\""))} else {r with _value: 0}))
+```
+
+For details on the =request syntax, see [SendEmail API v2 reference](https://docs.aws.amazon.com/ses/latest/APIReference-V2/API_SendEmail.html).
+
+{{% /tab-content %}}
+
+<!-------------------------------- BEGIN Mailjet ------------------------------->
+{{% tab-content %}}
+
+The example below uses the Mailjet Send API to send an alert email when more than 3 critical statuses since the last task run.
+
+{{% note %}}
+To view your Mailjet API credentials, sign in to Mailjet and open the [API Key Management page](https://app.mailjet.com/account/api_keys).
+{{% /note %}}
+
+```js
+import "http"
+
+// Import the Secrets package if you store your API keys as secrets.
+// For detail on how to do this, see Step 4 above.
+import "influxdata/influxdb/secrets"
+
+// Retrieve the secrets if applicable. Otherwise, skip this line
+// and add the API keys as Basic credentials in the Authorization header.
+MAILJET_APIKEY = secrets.get(key: "MAILJET_APIKEY")
+MAILJET_SECRET_APIKEY = secrets.get(key: "MAILJET_SECRET_APIKEY")
+
+numberOfCrits = from(bucket: "_monitoring")
+	|> range(start: -task.every)
+	|> filter(fn: (r) => (r.measurement == "statuses" and "r.level" == "crit")
+	|> count()
+
+numberOfCrits
+	|> map(fn: (r) => (if r._value > 3 then {
+    r with _value: http.post(url: "https://api.mailjet.com/v3.1/send",
+    headers: {"Content-type": "application/json", Authorization: "Basic ${MAILJET_APIKEY}:${MAILJET_SECRET_APIKEY}"},
+    data: bytes(v: "{
+        \"Messages\": [{
+          \"From\": {\"Email\": \”jane.doe@example.com\"},
+          \"To\": [{\"Email\": \"john.doe@example.com\"]},
+          \"Subject\": \”InfluxData critical alert\",
+          \"TextPart\": \”Example alert text\"
+          \"HTMLPart\":  `"<h3>Hello, to review critical alerts, review your <a href=\"https://www.example-dashboard.com/\">Critical Alert Dashboard</a></h3>}]}'
+              
+        }\""))} else {r with _value: 0}))
+```
+
+{{% /tab-content %}}
+
+<!-------------------------------- BEGIN Mailgun ---------------------------->
+
+{{% tab-content %}}
+
+The example below uses the Mailgun API to send an alert email when more than 3 critical statuses occur since the last task run.
+
+{{% note %}}
+To view your Mailgun API keys, sign in to Mailjet and open [Account Security - API security](https://app.mailgun.com/app/account/security/api_keys). <br> 
```

Comment:
Why the line break (`<br>`) here?

Reply:
At the time, I thought break long note w white space....but only added one and didn't look right...forgot to take out.